docs/class-tree.md

```diff
@@ -53,12 +61,13 @@ before rendering:
     import capellambse
 
     model = capellambse.MelodyModel("tests/data/ContextDiagram.aird")
-    diag = model.by_uuid("b7c7f442-377f-492c-90bf-331e66988bda").class_tree_diagram
+    diag = model.by_uuid("b7c7f442-377f-492c-90bf-331e66988bda").tree_diagram
     diag.render(
         "svgdiagram",
         edgeRouting="ORTHOGONAL",
         direction="Right",
         partitioning=False,
+        edgeLabelsSide="ALWAYS_DOWN",
```

Comment:
As there are quite a few options now, and they're all mentioned here: We should explicitly leave some out (with a comment hinting so), to make clear that it's not necessary to specify all of them every time.

It's kind of implied by the "default" hints in the text above, but who reads the explanation if there's a code example.